src/components/TierCard.js

```diff
@@ -154,6 +154,9 @@ class TierCard extends React.Component {
             .TierCard .fromCollectives.user:first-child {
               margin-left: 15px;
             }
+            .TiersPage {
```

Comment:
Why do we set the styles for the `TiersPage` in `TierCard` rather than `TiersPage`?